src/plugins/join/plugin.ts

```diff
@@ -1,5 +1,4 @@
 import BN from 'bn.js'
-import { BigNumber } from 'ethers/utils'
```

Comment:
@jellegerbrandy Are you sure the error is thrown from this import? What about the other imports of BigNumber?